Sources/KSCrashRecording/include/KSCrash.h

```diff
@@ -105,10 +105,24 @@ NS_ASSUME_NONNULL_BEGIN
 
 #pragma mark - API -
 
-/** Init KSCrash instance with custom base path. */
-- (instancetype) initWithBasePath:(NSString*) basePath;
+- (instancetype) init __attribute((unavailable("Use `sharedInstance` instead.")));
++ (instancetype) new __attribute((unavailable("Use `sharedInstance` instead.")));
+
+/**
+ * Specifies a custom base path for KSCrash installation.
+ * By default a "KSCrash" directory inside the default cache directory is used.
+ *
+ * @param basePath An absolute path to directory in which KSCrash stores the data.
+ *                 If `nil` the default directory is used.
+ *
+ * @note This method SHOULD be called before any use of `sharedInstance` method.
+ *       Any call of this method after that is ignored.
+ */
++ (void) setBasePath:(nullable NSString*) basePath;
```

Comment:
This means that user would try to change it after the installation. Maybe better move it to `KSCrashConfiguration` as a compulsory `init` parameter?

Reply:
Configuration only provided to `install`. And I can use `KSCrash` instance without calling `install` (e.g. to send reports).

Reply:
The dependency on the `sharedInstance` method call to determine whether the base path can be changed creates non-obvious behavior. It's not immediately clear to developers when they can or cannot change the base path. But I don't see a better solution without creating a separate instance of `KSCrash`.